uber/automated_emails.py

```diff
@@ -176,15 +401,19 @@ def days_before(days, dt, until=None):
                  needs_approval=False)
 
 MarketplaceEmail('Your {EVENT_NAME} Dealer registration is due in one week', 'dealers/payment_reminder.txt',
-                 lambda g: g.status == c.APPROVED and days_before(7, c.DEALER_PAYMENT_DUE, 2) and g.is_unpaid,
+                 lambda g: g.status == c.APPROVED and g.is_unpaid,
+                 when=days_before(7, c.DEALER_PAYMENT_DUE, 2),
                  needs_approval=False)
 
 MarketplaceEmail('Last chance to pay for your {EVENT_NAME} Dealer registration', 'dealers/payment_reminder.txt',
-                 lambda g: g.status == c.APPROVED and days_before(2, c.DEALER_PAYMENT_DUE) and g.is_unpaid,
+                 lambda g: g.status == c.APPROVED and g.is_unpaid,
+                 when=days_before(2, c.DEALER_PAYMENT_DUE),
                  needs_approval=False)
 
 MarketplaceEmail('{EVENT_NAME} Dealer waitlist has been exhausted', 'dealers/waitlist_closing.txt',
-                 lambda g: c.AFTER_DEALER_WAITLIST_CLOSED and g.status == c.WAITLISTED)
+                 lambda g: g.status == c.WAITLISTED,
+                 when=after(c.DEALER_WAITLIST_CLOSED),
+                 ident='uber_marketplace_waitlist_exhausted')
```

Comment:
Based on the above, I'm guessing you added idents to all of the emails which haven't already been approved for this year.  That seems like a good rule of thumb.

However, one thing to keep in mind is that I *think* I remember that you added the ability for us to deploy new code to past events, e.g. pushing the latest code to ``super2016.uber.magfest.org``.  As of this change, if we ever do that again, it'll probably blast out a ton of emails... unless we have email sending turned off for those old servers, which is probably a good idea if we haven't already done that.